test/infrastructure/docker/internal/docker/kind_manager.go

```diff
@@ -106,22 +103,22 @@ func (m *Manager) CreateWorkerNode(ctx context.Context, name, image, clusterName
 }
 
 // CreateExternalLoadBalancerNode will create a new container to act as the load balancer for external access.
-func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, port int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
+func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, hostPort, cpPort int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
```

Comment:
```suggestion
func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, hostPort int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
```
Let's drop cp port given that it is always 6443